glut/Makefile.am

```diff
@@ -67,8 +67,7 @@ endif
 
 test_glut_SOURCES=test_glut.c
 test_glut_LDADD=$(addprefix $(top_builddir)/, $(LIBS_MGH)) $(LIBS_VXL) -lz \
-	$(LIB_JPEG) $(LIB_TIFF)	$(X_LIBS) $(LIBS_OPENGL) $(top_builddir)/glut/libglut.a
-
+	$(LIB_JPEG) $(LIB_TIFF)	$(top_builddir)/glut/libglut.a $(X_LIBS) $(LIBS_OPENGL) 
```

Comment:
yeap -- as I have said somewhere -- it ignores 'TESTS' variable as far as I see it

my question to you (since I haven't trried yet locally, will do now) -- could you limit anyhow which tests get executed while doing it just straight in the shell